app/webpack/js/app/time_utils.test.js

```diff
@@ -0,0 +1,14 @@
+import { DateTime } from "luxon";
+
+import { parseTimeSlot } from "./time_utils";
+
+describe("time_utils", () => {
+  describe("parseTimeSlot", () => {
+    it("returns timeslots parsed into date times", () => {
+      expect(parseTimeSlot("1605135600::1605139200")).toEqual({
```

Comment:
I get why you had to change this in the db... but that is real ugly looking if you ever have to go through the raw data for whatever reason.

Reply:
imagine if those strings were iso timestamps ?!?  and then it did some weird stuff when it was "<iso>/<iso>" it made it "<iso>::<iso>".  I tried changing the coder for the `store` to JSON and then it something equally weird but different.  so ... ints are easy to see as strings so ... simple.